README.md

```diff
@@ -60,6 +60,23 @@ Specific instructions how to run the containers are stored within the same direc
     In case you are using [project "Piper"](https://sap.github.io/jenkins-library/) you can configure certain steps 
     to use docker images instead of the local Jenkins environment. These steps will automatically pull and run these 
     images.
+ 
+### Setting up Jenkins Server
+The `cx-server` is a toolkit that is developed to manage the lifecycle of the Jenkins server.
+In order to use the toolkit, you need a file named `cx-server` and a configuration file `server.cfg`. 
+You can generate these files using the docker command
+
+```sh
+docker run -it --rm -u $(id -u):$(id -g) -v ${PWD}:/cx-server/mount/ ppiper/cxserver-companion:latest init-cx-server
```

Comment:
trying to reproduce this command with a locally built copy of the image, but it fails

```
docker run -it --rm -u $(id -u):$(id -g) -v ${PWD}:/cx-server/mount/ ppiper/cxserver-companion:latest init-cx-server
docker: Error response from daemon: OCI runtime create failed: container_linux.go:344: starting container process caused "exec: \"init-cx-server\": executable file not found in $PATH": unknown.
```

what is wrong?